.github/workflows/release.yml

```diff
@@ -284,3 +284,47 @@ jobs:
         run: |
           docker push binpash/hadoop-submit:${{ env.RELEASE }}
           docker push binpash/hadoop-submit:latest
+
+  run-containers-and-tests:
```

Comment:
Why would it need to run every night? New images are pushed to the registry only after a release, so the canary test pulls the latest image, executes it, and checks if everything works correctly. If it passes, it means the image is valid, and there’s no need for daily tests. Am I missing something?

Reply:
I was thinking something similar to this (https://github.com/binpash/try/blob/main/.github/workflows/test.yaml). I think it can run after the rest, but also run every night to make sure that nothing breaks even between pushes. @mgree do you have thoughts on that? Do you think this is not relevant here even though it is for try?

Reply:
I don't have much context, but it makes sense to run things frequently if there are dependencies which could cause breakage. I tend to go for weekly canaries.

Reply:
@mohamadaminkarami could you modify the canary to run as it does now but also scheduled to run once a week?

Reply:
I created a new workflow file that schedules canary tests to run weekly.

Reply:
Great thanks a lot!